### PR TITLE
Parser for OSP Neutron metadata-agent.log

### PR DESCRIPTION
**Pull request by ppandey-redhat**
Trello: https://trello.com/c/CaBVYpV2

### CODE REVIEW

**Comment by lhuett** [comment]
Looks good pending Spec approval.

File: insights/parsers/tests/test_neutron_metadata_agent_log.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+METADATA_AGENT_LOG = """
+2018-06-08 17:29:55.894 11770 WARNING neutron.agent.metadata.agent [-] Server does not support metadata RPC, fallback to using neutron client
+2018-06-08 17:29:55.907 11770 ERROR neutron.agent.metadata.agent [-] Unexpected error.
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent Traceback (most recent call last):
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent   File "/usr/lib/python2.7/site-packages/neutron/agent/metadata/agent.py", line 109, in __call__
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent     self._authenticate_keystone()
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent   File "/usr/lib/python2.7/site-packages/neutronclient/client.py", line 218, in _authenticate_keystone
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent     raise exceptions.Unauthorized(message=resp_body)
+2018-06-08 17:29:56.126 11770 TRACE neutron.agent.metadata.agent Unauthorized: {"error": {"message": "The resource could not be found.", "code": 404, "title": "Not Found"}}
+"""
```
Comment (psachin) [inline insights/parsers/tests/test_neutron_metadata_agent_log.py]: Please append `.strip()`

Comment (ppandey-redhat) [reply insights/parsers/tests/test_neutron_metadata_agent_log.py]: Acknowledged, changes done in the latest patch.

File: insights/specs/default.py [hunk 2]
```diff
@@ -379,6 +379,7 @@ def semid(broker):
     networkmanager_dispatcher_d = glob_file("/etc/NetworkManager/dispatcher.d/*-dhclient")
     neutron_conf = simple_file("/etc/neutron/neutron.conf")
     neutron_l3_agent_log = simple_file("/var/log/neutron/l3-agent.log")
+    neutron_metadata_agent_log = simple_file("/var/log/neutron/metadata-agent.log")
```
[the remark this answers is not in the document]
Comment (ppandey-redhat) [reply insights/specs/default.py]: Acknowledged, changes done in the latest patch.

**Comment by xiangce** [comment]
test this please

**Comment by ppandey-redhat** [comment]
@xiangce 
I have already tested it. To check why CI integration default tests is getting stuck, I have added few commits but even then the default tests are getting stuck at the same point. Upto commit "Resolved conflicts with neutron_metadata_agent_conf" the tests were getting passed.

**Pull request merged**